adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -352,13 +343,14 @@ def shake(self, shake_threshold=30):
         """
         return self._lis3dh.shake(shake_threshold=shake_threshold)
 
-    def _touch(self, i):
-        if not isinstance(self._touches[i], touchio.TouchIn):
-            # First time referenced. Get the pin from the slot for this touch
-            # and replace it with a TouchIn object for the pin.
-            self._touches[i] = touchio.TouchIn(self._touches[i])
-            self._touches[i].threshold += self._touch_threshold_adjustment
-        return self._touches[i].value
+    def _touch(self, pin):
+        touchin = self._touches.get(pin)
+        if not touchin:
+            # First time referenced. Make TouchIn object for the pin
+            touchin = touchio.TouchIn(pin)
+            touchin.threshold += self._touch_threshold_adjustment
+            self._touches[pin] = touchin
+        return self._touches[pin].value
```

Comment:
Use the assigned value instead of looking it up again in the dict.
```suggestion
        return touchin.value
```

Reply:
Whoops, good eye!